proto/node/create/v1/node_instance_creation.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
```

Comment:
We can move the file in: `proto/nodeinstance/v1/creation.proto`, in order to be consistent with other files.